libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -739,19 +761,25 @@ ParseFlags parseFlag(CompileCommand cmd, const CompileCommandFilter flag_filter,
                 st = State.keep;
             } else if (st == State.pathArgumentToAbsolute) {
                 st = State.keep;
-                auto p = buildNormalizedPath(workdir, arg).absolutePath;
-                rval.put(p);
-                includes.put(p);
+                putNormalizedAbsolute(arg);
             } else if (st == State.priorityKeepNextArg) {
                 st = State.keep;
                 rval.put(arg);
+	    } else if (st == State.checkingForQuotation) {
+		path.put(" " ~ arg);
+		if (isEndingWithQuotation(arg)) {
+		  st = State.keep;
+		  putNormalizedAbsolute(path.data.strip('"'));
```

Comment:
The quotation mark should not be added to path. Internally and when executing external programs it isn't needed anymore. It is just "there" for e.g. bash expansion etc.